apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/NormalizedCache.java

```diff
@@ -40,21 +45,24 @@ protected RecordFieldAdapter recordAdapter() {
   }
 
   /**
-   * @param key The key of the record to read
+   * @param key          The key of the record to read.
+   * @param cacheHeaders The cache headers associated with the request which generated this record.
    * @return The {@link Record} for key. If not present return null.
    */
-  @Nullable public abstract Record loadRecord(String key);
+  @Nullable public abstract Record loadRecord(@Nonnull String key, @Nonnull CacheHeaders cacheHeaders);
```

Comment:
Should we have overloaded version of this function without argument `CacheHeaders`?